apps/list.c

```diff
@@ -277,6 +279,63 @@ static void list_kdfs(void)
     sk_EVP_KDF_pop_free(kdfs, EVP_KDF_free);
 }
 
+/*
+ * Serializers
+ */
+DEFINE_STACK_OF(OSSL_SERIALIZER)
+static int serializer_cmp(const OSSL_SERIALIZER * const *a,
+                          const OSSL_SERIALIZER * const *b)
+{
+    int ret = OSSL_SERIALIZER_number(*a) - OSSL_SERIALIZER_number(*b);
+
+    if (ret == 0)
+        ret = strcmp(OSSL_PROVIDER_name(OSSL_SERIALIZER_provider(*a)),
+                     OSSL_PROVIDER_name(OSSL_SERIALIZER_provider(*b)));
+
+    return ret;
+}
+
+static void collect_serializers(OSSL_SERIALIZER *serializer, void *stack)
+{
+    STACK_OF(OSSL_SERIALIZER) *serializer_stack = stack;
+
+    sk_OSSL_SERIALIZER_push(serializer_stack, serializer);
+    OSSL_SERIALIZER_up_ref(serializer);
+}
+
+static void list_serializers(void)
+{
+    STACK_OF(OSSL_SERIALIZER) *serializers;
+    int i;
+
+    serializers = sk_OSSL_SERIALIZER_new(serializer_cmp);
```

Comment:
add a null check here..

Reply:
Added NULL checks for all the cases above too.  There were none.